ethereumkit/src/main/java/org/p2p/ethereumkit/external/api/response/TokenBalancesResponse.kt

```diff
@@ -0,0 +1,30 @@
+package org.p2p.ethereumkit.external.api.response
+
+import com.google.gson.annotations.SerializedName
+import org.p2p.ethereumkit.internal.models.EthAddress
+import java.math.BigInteger
+
+internal data class TokenBalancesResponse(
+    @SerializedName("address")
+    val address: EthAddress? = null,
```

Comment:
EthAddressResponse

Reply:
no, it's not response, it's wrapped primitive class